secp256k1-sys/build.rs

```diff
@@ -20,11 +20,16 @@ fn main() {
                .include("depend/secp256k1/include")
                .include("depend/secp256k1/src")
                .flag_if_supported("-Wno-unused-function") // some ecmult stuff is defined but not used upstream
+               .flag_if_supported("-Wno-unused-parameter") // patching out printf causes this warning
```

Comment:
I think this can be solved more locally but I haven't used C for a while so I don't remember how.

Reply:
There's no standard C way to do it. Any specific compiler would have annotations but they wouldn't work for others. (Of course, `flag_if_supported` also not universal. But `cc` makes an effort to translate it for the common compilers.)

And in any case, if I wanted a more local solution, I'd just patch out the offending methods entirely :).

Reply:
I mean something like using `(void)x` works IIRC which can be used in the define. But I don't know how to apply it to multiple arguments.